go/vt/mysqlctl/schema.go

```diff
@@ -46,6 +47,15 @@ func (mysqld *Mysqld) executeSchemaCommands(sql string) error {
 	return mysqld.executeMysqlScript(params, strings.NewReader(sql))
 }
 
+// tableList returns an IN clause "('t1', 't2'...) for a list of tables."
+func tableListSql(tables []string) string {
+	if len(tables) == 0 {
+		return "()"
```

Comment:
It's an invalid SQL syntax to query `WHERE IN ()` -- I suggest returning an empty string+error

Reply:
We'll also need a test to prove that it works correctly.

Reply:
i've added errors for 0 tables, though the code should never be in that state. also i'm escaping table names using the sql encoder.